src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -801,7 +844,7 @@ def _execute_simulation(
                 self._simulation_finish_actions(slot_count)
                 return
 
-            self._results.update_csv_on_market_cycle(slot_no, self.area)
+            # self._results.update_csv_on_market_cycle(slot_no, self.area)
```

Comment:
Yes, because the simulation results are not adapted yet (will do it in the context of this https://gridsingularity.atlassian.net/browse/GSYE-258), generating the CSV files caused the simulation to crash thus not allowing me to test whether the simulation is running correctly. I'll reintroduce it at the next PR where the simulation results are dealt with.